pypika/terms.py

```diff
@@ -1060,6 +1061,25 @@ def tables_(self):
         return self.term.tables_
 
 
+class CustomFunction():
+    def __init__(self, term, params=None):
+        self.term = term
+        self.params = params
+
+    def __call__(self, *args, **kwargs):
+        if (self.params is None):
```

Comment:
this is minor, but we generally omit the params in if statements in other parts of the code

Reply:
I don't think that I got this one. What do you mean?

Reply:
Can you review if was it that you meant?